openedx/adg/lms/student/views.py

```diff
@@ -124,10 +128,37 @@ def get_context_data(self, **kwargs):
         """
         context = super(DashboardView, self).get_context_data(**kwargs)
 
+        platform_name = configuration_helpers.get_value('platform_name', settings.PLATFORM_NAME)
+        activation_email_support_link = configuration_helpers.get_value(
+            'ACTIVATION_EMAIL_SUPPORT_LINK', settings.ACTIVATION_EMAIL_SUPPORT_LINK
+        ) or settings.SUPPORT_SITE_LINK
```

Comment:
We can move these into the if condition block as these variables are only being used inside the if condition block.